.github/CODEOWNERS

```
@@ -147,6 +147,8 @@ Lib/ast.py                    @isidentical
 **/*cgi*                      @ethanfurman
 **/*tarfile*                  @ethanfurman
 
+**/*tomllib*                  @hukkin @encukou
```

This file can only list core devs, sadly.
```suggestion
**/*tomllib*                  @encukou
```